core/trino-main/src/main/java/io/trino/execution/buffer/OutputBuffers.java

```diff
@@ -249,6 +249,13 @@ public OutputBuffers withNoMoreBufferIds()
         return new OutputBuffers(type, version + 1, true, buffers, exchangeSinkInstanceHandle);
     }
 
+    public OutputBuffers withExchangeSinkInstanceHandle(ExchangeSinkInstanceHandle updatedExchangeSinkInstanceHandle)
```

Comment:
withUpdatedExchangeSinkInstanceHandle?

Reply:
this is not called yet. To be added separatelly?

Reply:
Yeah, let me drop it for now